### ISSUE TITLE
Quantitative piechart shows bins that are outside of the selected range

### ISSUE THREAD

**Issue by fedorov** [ImagingDataCommons/IDC-WebApp#377]
Slider has range [43,62], but the piechart includes values outside the range.

![image](https://user-images.githubusercontent.com/313942/95391431-0cb99300-08c5-11eb-907a-edcfdd9ed212.png)

**Comment by fedorov** [comment]
Also, the numbers do not sum up to 18, which is reportedly the number of cases in my cohort.

**Comment by s-paquette** [comment]
Since filters are applied at the study level, that would be why. We can only fix that by applying at the instance level.

**Comment by s-paquette** [comment]
They won't add up for the same reason--there's a Venn diagram effect going on that can't be seen.

**Comment by fedorov** [comment]
Yes, most likely that is the reason. I always forget about this. Would be good to check that indeed this is the case. Maybe @madelyngreyes can check this against the quantitative measurements view.

**Comment by G-White-ISB** [comment]
This is explained by the tooltip text.

**Comment by fedorov** [comment]
I rephrased the issue. I think we need to confirm that the information shown in the piecharts is accurate using the BigQuery tables, since it is impossible to confirm this using the portal.

**Comment by G-White-ISB** [comment]
We've had some discussion about this at ISB. The API is going to pull data directly from Big Query tables. I believe we will make the actual queries used to fetch the data available. Then the web app counts can be verified by checking against the API results. This can be done as a user check, but we will also write test suites to do this.

**Comment by fedorov** [comment]
Sounds good, then there is no need for Madelyn to look into this.

**Comment by wlongabaugh** [comment]
Link to public view that can be used to display the cohort as defined by the UI:

https://console.cloud.google.com/bigquery?authuser=2&project=canceridc&p=canceridc-data&d=idc_views&t=dicom_pivot_wave1&page=table

**Comment by wlongabaugh** [comment]
Providing the query is covering in issue #182

**Comment by wlongabaugh** [comment]
Testing will be needed after #182 is addressed

**Comment by G-White-ISB** [comment]
It might be time to do this

**Comment by G-White-ISB** [comment]
related to #182. BQ can now be used to check the pie chart values
